### PR TITLE
Removed Extra Space after 'Share'

### LINKED ISSUE #4611: Remove the extra space after `share`
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file `app/views/editor/rich.html.erb` in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 
https://github.com/publiclab/plots2/blob/87db5642c4f434e9eaaa3478f79d2584916c6d09/app/views/editor/rich.html.erb#L30-L37

Remove the extra space which is below the `share` button text.

screenshot 152

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
- <div class="ple-module container"> 
-
-  <div class="col-md-3"></div> 
-  <div class="ple-errors col-md-9"></div> 

+  <div class="ple-errors"></div> 
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by JAIDEV123**
Fixes Issue : https://github.com/publiclab/plots2/issues/4611

Fixes #4611(<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/editor/rich.html.erb [hunk 1]
```diff
@@ -29,8 +29,6 @@
 
     <!-- errors module -->
     <div class="ple-module container">
```
Comment (oorjitchowdhary) [inline app/views/editor/rich.html.erb]:
```diff
- <div class="ple-module container">
```
Remove this line too

File: app/views/editor/rich.html.erb [hunk 2]
```diff
@@ -29,8 +29,6 @@
...
-
-      <div class="col-md-3"></div>
       <div class="ple-errors col-md-9"></div>
```
Comment (oorjitchowdhary) [inline app/views/editor/rich.html.erb]:
```suggestion
      <div class="ple-errors"></div>
```
Do this..

**Comment by oorjitchowdhary** [comment]
Hi @JAIDEV123, welcome to Public Lab! 
I've suggested some changes in the code. Please make them.

Also, upload a screenshot of the feature working on your localhost
Thanks

File: app/views/editor/rich.html.erb (hunk 1 above)
Comment (JAIDEV123) [reply app/views/editor/rich.html.erb]: All right. Removing the closing `div ` as well

**Comment by JAIDEV123** [comment]
@publiclab/reviewers Hey! Currently, it seems like I have to create a new PR in order to add the suggested changes proposed by @oorjitchowdhary . Is there another way? (instead of closing this PR and then starting another one). Also, how can I run this on my PC?

Thanks!

**Comment by oorjitchowdhary** [comment]
Hey, you can still make changes in this PR. You opened this PR from the `patch-1` branch of your forked repo.. You can go to that branch (JAIDEV123/plots2:patch-1) and do the changes..

**Comment by JAIDEV123** [comment]
Hey!

Looks like all checks have passed :)

**Comment by oorjitchowdhary** [comment]
Hi @JAIDEV123, this looks great. We just require screenshots now.
You can setup plots2 repo on your localhost through the steps given in README.md
Thanks

**Comment by JAIDEV123** [comment]
Seems to have hit a roadblock. 

screenshot 4

I've tried bundle install and bundle update with no progress. Tried rolling back ruby to 2.4 but error message states that I have to use 2.5 or later. Do you know of a fix for this? Could this be because patch-1 branch hasn't been updated in some time?

**Comment by jywarren** [comment]
Hi! Would you like to take a look at this page to see if you can get some
setup help? https://github.com/publiclab/plots2/issues/3840

I think though that you may need to install Ruby 2.4.4. You could do this
using RVM -- http://rvm.io/

**Comment by JAIDEV123** [comment]
Hey. It looks like I'm going to be busy for a while. Could someone else please continue this?

**Comment by gauravano** [comment]
Post page view:

editor_view

I think, we are not rendering anything inside deleted div, right? 
@rexagod @jywarren please give one more review for this to merge. Thanks!

**Comment by gauravano** [comment]
Merged :tada: :balloon: !

Hi @JAIDEV123, thanks for the nice work. We would love to have more help from you. If you are interested in solving more issues then check here - https://code.publiclab.org/#r=all for a new issue. In case, you are unable to find an issue, feel free to comment here. Thanks!

**Pull request merged**